messaging/layers/transport/filters_manager.go

```diff
@@ -127,13 +129,15 @@ func (f *FiltersManager) Init(
 type FiltersToInitialize struct {
 	ChatID      string
 	PubsubTopic string
+	// TODO (#6384) temporary flag while migrating community shards
+	IsCommunity bool
```

Comment:
```suggestion
	DistinctByPubsub bool
```

Having the `community` keyword in the transport layer is something we should avoid. I believe `DistinctByPubsub` conveys the intention more precisely.